osu.Game/Overlays/Profile/Header/BannerHeaderContainer.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Threading;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Online.API.Requests.Responses;
+using osu.Game.Overlays.Profile.Header.Components;
+
+namespace osu.Game.Overlays.Profile.Header
+{
+    public partial class BannerHeaderContainer : CompositeDrawable
+    {
+        public readonly Bindable<UserProfileData?> User = new Bindable<UserProfileData?>();
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            Alpha = 0;
+            RelativeSizeAxes = Axes.Both;
+            FillMode = FillMode.Fit;
```

Comment:
The fact this works gets me every time. Like specifying a relative height but actually having it decided by the fill mode / ratio.